app/models/manageiq/providers/hawkular/middleware_manager/alert_manager.rb

```diff
@@ -0,0 +1,85 @@
+module ManageIQ::Providers
+  class Hawkular::MiddlewareManager::AlertManager
+    require 'hawkular/hawkular_client'
+
+    def initialize(ems)
+      @alerts_client = ems.connect.alerts
```

Comment:
@lucasponce Would using `ems.alerts_client` here (and in alert_profile_manager) be equivalent?
I don't have a strong preference, just wonder if there is some subtle distiction between this and how event catcher (which calls `alerts_client`) gets its connection.